recipes/autoconf-archive/all/test_package/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import contextlib
+import os
+import shutil
+
+required_conan_version = ">=1.36.0"
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = "configure.ac", "Makefile.am", "test_package.c"
+    test_type = "build_requires"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def build_requirements(self):
+        self.build_requires("automake/1.16.3")
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
```

Comment:
(Thinking aloud, because I haven't used this recipe, so I cannot tell if it is always useful or not -> ) Does it make sense to add some of these build-requirements as regular requirements of `autoconf-archive`? If there are always needed when you want to use `autoconf-archive` it might be useful if they are also requirements.

Reply:
Nice idea, but not applicable in this case.
`autoconf-archive` is a collection of input scripts/data for m4/automake.

`automake`/`autoconf` are a collection of perl+bash scripts.
It is perhaps more meaningful to add the requirement to `autoconf``.

Reply:
I guess it's theoretically possible for this to be useful without automake/autoconf. It's better as it is, as a 'plugin' for autoconf.